corehq/form_processor/models.py

```diff
@@ -563,6 +563,11 @@ def is_deleted(self):
     def dynamic_case_properties(self):
         return OrderedDict(sorted(self.case_json.iteritems()))
 
+    def get_json(self, lite=False):
```

Comment:
yeah this is gross i agree. `get_json` is one of those generic function names that gets called on `self` a lot so it can be sometimes hard to tell if you're replacing the correct `get_json`. originally punted because of that, but seems reasonable and worth the effort to change. will update